R/teal_slice-store.R

```diff
@@ -0,0 +1,59 @@
+#' Store teal_slices object to a file
+#'
+#' This function takes a teal_slices object and saves it to a file in `JSON` format.
+#' The teal_slices object contains information about filter states and can be used to
+#' create, modify, and delete filter states. The saved file can be later loaded using
+#' the `slices_restore` function.
+#'
+#' @param tss (`teal_slices`) object to be stored.
+#' @param file (`character(1)`) The file path where `teal_slices` object will be saved.
+#'  The file extension should be `".json"`.
+#'
+#' @return None
+#'
+#' @examples
+#' \dontrun{
```

Comment:
`\dontrun` tags will be ignored on CRAN (if we ever get there). Perhaps put the store example under `if (interactive())`?